updating/updating_a_cluster/updating-cluster-web-console.adoc

```diff
@@ -19,49 +19,39 @@ You can update, or upgrade, an {product-title} cluster by using the web console.
-* Review the list of APIs that were removed in Kubernetes 1.27, migrate any affected components to use the new API version, and provide the administrator acknowledgment. For more information, see xref:../../updating/preparing_for_updates/updating-cluster-prepare.adoc#updating-cluster-prepare[Preparing to update to {product-title} 4.14].
-* If you run an Operator or you have configured any application with the pod disruption budget, you might experience an interruption during the update process. If `minAvailable` is set to 1 in `PodDisruptionBudget`, the nodes are drained to apply pending machine configs which might block the eviction process. If several nodes are rebooted, all the pods might run on only one node, and the `PodDisruptionBudget` field can prevent the node drain.
+include::modules/before-updating-web-console.adoc[leveloffset=+1]
 
 [IMPORTANT]
 ====
 * When an update is failing to complete, the Cluster Version Operator (CVO) reports the status of any blocking components while attempting to reconcile the update. Rolling your cluster back to a previous version is not supported.  If your update is failing to complete, contact Red Hat support.
 * Using the `unsupportedConfigOverrides` section to modify the configuration of an Operator is unsupported and might block cluster updates. You must remove this setting before you can update your cluster.
 ====
 
+include::modules/update-changing-update-server-web.adoc[leveloffset=+1]
+
 [role="_additional-resources"]
 .Additional resources
 
-* xref:../../architecture/architecture-installation.adoc#unmanaged-operators_architecture-installation[Support policy for unmanaged Operators]
-
-include::modules/update-using-custom-machine-config-pools-canary.adoc[leveloffset=+1]
-
-If you want to use the canary rollout update process, see xref:../../updating/updating_a_cluster/update-using-custom-machine-config-pools.adoc#update-using-custom-machine-config-pools[Performing a canary rollout update].
+* xref:../../updating/understanding_updates/understanding-update-channels-release.adoc#understanding-update-channels-releases[Understanding update channels and releases]
 
 include::modules/machine-health-checks-pausing-web-console.adoc[leveloffset=+1]
 
-include::modules/updating-sno.adoc[leveloffset=+1]
+include::modules/update-upgrading-web.adoc[leveloffset=+1]
+
+include::modules/update-conditional-web-console.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
 
-* For information on which machine configuration changes require a reboot, see the note in xref:../../architecture/control-plane.adoc#about-machine-config-operator_control-plane[About the Machine Config Operator].
+* xref:../../updating/updating_a_cluster/updating-cluster-cli.adoc#update-conditional-upgrade-pathupdating-cluster-cli[Updating along a conditional update path]
```

Comment:
I don't understand why we link to the CLI here.  Can we link like:

```asciidoc
* xref:../../updating/understanding_updates/understanding-update-channels-release.adoc#conditional-updates-overview_understanding-update-channels-releases[Update recommendations and Conditional Updates]
```

or some such to link to the understanding section on conditional updates?